src/platform-integration/android/platform-views.md

```diff
@@ -406,6 +406,12 @@ android {
 }
 ```
 
+#### Multi threaded draw calls
+
+When the Android View draws using a separate thread,
```

Comment:
This requirement doesn't come from having a background thread render the view but from the views themselves. I would rephrase:

Certain Android Views do not invalidate themselves when their content changes. Some example views include `SurfaceView` and `SurfaceTexture`. When your Platform View includes these views you are required to manually invalidate the view after they have been drawn to (or more specifically: after the swap chain is flipped). Manual view invalidation is done by calling `invalidate` on the View or one of its parent views.

Reply:
done